elements/castbar.lua

```diff
@@ -183,6 +188,11 @@ local UNIT_SPELLCAST_INTERRUPTED = function(self, event, unit, spellname, _, cas
 		return
 	end
 
+	local text = castbar.Text
+	if(text) then
+		text:SetText(INTERRUPTED)
+	end
+
 	castbar.casting = nil
```

Comment:
The user probably won't even notice that if they don't use the .holdTime option because the bar will be hidden by the OnUpdate script on the next frame. The change allows to use the same PostUpdate hook for UNIT_SPELLCAST_FAILED and _INTERRUPTED.

I would even go further and set .holdTime to `GetTime() + a layout defined constant` so that it would be possible to eliminate the necessity for a PostUpdate hook (unless they want coloring or something)

I'm not sure about setting the bar value to its max though. While this is the default ui behavior, maybe it is interesting to see when the cast was interrupted.